Maquetado/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
We use **two spaces** for indentation. In client projects there will a predefined prettify and linter configuration that handles styling, Even if it is not necessary right now you could configure a prettier on your editor and set it the indentation to two spaces so you dont have to be bothered on the training with this issue.